open_dread_rando/dread_patcher.py

```diff
@@ -30,6 +30,7 @@ def create_custom_init(editor: PatcherEditor, configuration: dict):
     inventory: dict[str, int] = configuration["starting_items"]
     starting_location: dict = configuration["starting_location"]
     starting_text: list[list[str]] = configuration.get("starting_text", [])
+    shareable_hash: str = configuration["shareable_hash"]
```

Comment:
I wouldn't call this `shareable_hash`, that only makes sense in the context of Randovania.
This doesn't even need to have `hash` or anything in the name: it's purely an identifier that the save file must match the mod.